code/game/machinery/_machinery.dm

```diff
@@ -119,10 +119,9 @@ Class Procs:
 	var/critical_machine = FALSE //If this machine is critical to station operation and should have the area be excempted from power failures.
 	var/list/occupant_typecache //if set, turned into typecache in Initialize, other wise, defaults to mob/living typecache
 	var/atom/movable/occupant = null
-	var/unsecuring_tool = /obj/item/wrench
```

Comment:
literally nothing but one other location references this var. Why is it here?

Reply:
It was here from the times new machinery construction/deconstruction was added, but it wasn't used.